From b762b2d9aab325905c2b3bb93f418c95314a7b65 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Wed, 3 Apr 2019 11:57:06 +0200
Subject: [PATCH] PCI: Fix compile warning about unused functions

pci_bus_lock() and pci_bus_unlock() are not used in this functions and defined
static, this looks like some left over parts from the old PCIe driver, the new
one does not need them.
Also add the export to make this file now exactly match the unmodified file
from kernel 4.9.161.
---
 drivers/pci/search.c | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/drivers/pci/search.c b/drivers/pci/search.c
old mode 100755
new mode 100644
index 98cca0ad57c1..33e0f033a48e
--- a/drivers/pci/search.c
+++ b/drivers/pci/search.c
@@ -16,18 +16,6 @@
 DECLARE_RWSEM(pci_bus_sem);
 EXPORT_SYMBOL_GPL(pci_bus_sem);
 
-static void pci_bus_lock(void)
-{
-	WARN_ON(in_interrupt());
-	down_read(&pci_bus_sem);
-}
-
-static void pci_bus_unlock(void)
-{
-	WARN_ON(in_interrupt());
-	up_read(&pci_bus_sem);
-}
-
 /*
  * pci_for_each_dma_alias - Iterate over DMA aliases for a device
  * @pdev: starting downstream device
@@ -182,6 +170,7 @@ struct pci_bus *pci_find_next_bus(const struct pci_bus *from)
 	up_read(&pci_bus_sem);
 	return b;
 }
+EXPORT_SYMBOL(pci_find_next_bus);
 
 /**
  * pci_get_slot - locate PCI device for a given PCI slot
